lib/msf/core/module/reference.rb

```diff
@@ -120,6 +133,25 @@ def initialize(in_ctx_id = 'Unknown', in_ctx_val = '')
       self.site = "Logo: #{in_ctx_val}"
     elsif in_ctx_id == 'SOUNDTRACK'
       self.site = "Soundtrack: #{in_ctx_val}"
+    elsif in_ctx_id == 'ATT&CK'
+      # Handle sub-technique IDs correctly so they render URL in the correct format
+      # Example: T1218.011 becomes T1218/011
+      match = in_ctx_val.match(/\A(?<technique>T\d{4})\.(?<sub_technique>\d{3})\z/.freeze)
```

Comment:
Instead of this special casing, would it make sense to make it more generic to future proof things? 🤔 

i.e. Grabbing the a-z at the start to map to category, and just replacing `.` with `/`